bindings/c/test/unit/unit_tests.cpp

```diff
@@ -876,23 +876,29 @@ TEST_CASE("fdb_transaction_set_read_version future_version") {
 const std::string EMPTY = Tuple().pack().toString();
 const KeyRef RECORD = "RECORD"_sr;
 const KeyRef INDEX = "INDEX"_sr;
-static KeyRef primaryKey(const int i) {
-	return KeyRef(format("primary-key-of-record-%08d", i));
+static Key primaryKey(const int i) {
+	return Key(format("primary-key-of-record-%08d", i));
 }
-static KeyRef indexKey(const int i) {
-	return KeyRef(format("index-key-of-record-%08d", i));
+static Key indexKey(const int i) {
+	return Key(format("index-key-of-record-%08d", i));
 }
-static ValueRef dataOfRecord(const int i) {
-	return KeyRef(format("data-of-record-%08d", i));
+static Value dataOfRecord(const int i) {
+	return Value(format("data-of-record-%08d", i));
 }
 static std::string indexEntryKey(const int i) {
-	return Tuple().append(prefix).append(INDEX).append(indexKey(i)).append(primaryKey(i)).pack().toString();
+	return Tuple()
+	    .append(StringRef(prefix))
+	    .append(INDEX)
+	    .append(indexKey(i).contents())
+	    .append(primaryKey(i).contents())
+	    .pack()
+	    .toString();
 }
 static std::string recordKey(const int i) {
 	return Tuple().append(prefix).append(RECORD).append(primaryKey(i)).pack().toString();
 }
 static std::string recordValue(const int i) {
-	return Tuple().append(dataOfRecord(i)).pack().toString();
+	return Tuple().append(dataOfRecord(i).contents()).pack().toString();
```

Comment:
Is there any reason explicitly converting standalone to ref? Is that a recommended practice?

Reply:
It didn't compile without it this. I'm not sure exactly why but I'm guessing it has to do with `Tuple::append` being overloaded

Reply:
Interesting. This https://github.com/apple/foundationdb/pull/5965/files#diff-2cefc54d5914b856d142ee213547f98703018a7286fadb55f4e59871b35fecc6R898 looks working without the conversion.

Reply:
Thanks - it looks like I was mistaken